common/lifecyleMgr.go

```diff
@@ -590,6 +591,15 @@ func (lcm *lifecycleMgr) IsForceLoggingDisabled() bool {
 	return lcm.disableSyslog
 }
 
+func (lcm *lifecycleMgr) DownloadToTempPath() bool {
+	ret, err := strconv.ParseBool(lcm.GetEnvironmentVariable(EEnvironmentVariable.DownloadToTempPath()))
```

Comment:
consider doing what SetForceLogging does above, and save the parsed result rather than parsing it every time.